templates/tfengine/components/project/gke_clusters/main.tf

```diff
@@ -1,3 +1,19 @@
+/**
```

Comment:
Have you tried running the engine on the 'full.yaml' example? I believe this won't output correctly.

Reply:
I'll try it. Why do you think it won't output correctly? Is this invalid syntax for .tf files?

Reply:
You're right, it doesn't show up right. What adds this header when generating the output files?

Reply:
Changed it to only check Go files for now. I think the double license headers happen because we combine .tf files to produce the final output.

Reply:
It gets added by the terragrunt leaf component